doc/tor.md

```diff
@@ -211,3 +209,96 @@ for normal IPv4/IPv6 communication, use:
   Otherwise it is trivial to link them, which may reduce privacy. Onion
   services created automatically (as in section 2) always have only one port
   open.
+
+- Some states have the ability to monitor and block use of the Tor network.
+  Recent examples include China and India. If you are in an environment that
```

Comment:
nit: "Recent examples include China and India."

I'd rather take out specific examples of countries to make it softer. I don't think we want to be unnecessarily antagonistic to national governments for no upside. The priority for this PR is that users have access to good documentation on how to use Tor rather than documenting where those users likely are.

Reply:
Removed